tests/test_widgets.py

```diff
@@ -21,7 +21,7 @@ class TestHTMLParams:
     def test_basic(self):
         assert html_params(foo=9, k="wuuu") == 'foo="9" k="wuuu"'
         assert html_params(class_="foo") == 'class="foo"'
-        assert html_params(class__="foo") == 'class_="foo"'
```

Comment:
Didn't realize there was a test for this. Maybe it's better to just strip one trailing underscore? What do you think?

Reply:
Honnestly I am not sure I understand the need for being able to set a `class_` attribute.
I originally tried to keep up with the existing tests, but then I thought that your `rstrip` suggestion was fine.
If suggest we merge as it is, and if someone shows up asking why `class__` does not become `class_` anymore, I will fix it.